build/cmake/FindSndFile.cmake

```diff
@@ -1,6 +1,6 @@
 include(GetPlatformInfo)
 
-if (OS_IS_WIN)
+if (OS_IS_WIN AND (NOT MINGW))
     find_path(SNDFILE_INCDIR sndfile.h PATHS ${PROJECT_SOURCE_DIR}/dependencies/include;)
     if (MINGW)
```

Comment:
wouldn't this then be dead code?

Reply:
Originally the path was took for all flavors of windows. This narrows to a subset that does not include mingw.

Reply:
But causes sndfile to not get found

Reply:
I see what you mean with dead code, I've removed the mingw stuff inside the windows path that is avoided on mingw.